sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -587,6 +587,11 @@ namespace Azure { namespace Storage { namespace Blobs {
      * in this option. Default is to replace.
      */
     Models::BlobCopySourceTagsMode CopySourceTagsMode;
+
+    /**
+     * @brief Optional. Source authentication used to access the source blob.
+     */
+    Azure::Nullable<std::string> SourceAuthentication;
```

Comment:
according to this discussion, we should call this `SourceAuthorization`.

https://github.com/Azure/azure-sdk-for-cpp/issues/4718